src/Components/SmartComponents/ClusterDetail/ClusterDetailPage.js

```diff
@@ -7,12 +7,18 @@ import { useSelector } from 'react-redux';
 import ErrorHandler from '../../PresentationalComponents/ErrorHandler';
 import useChrome from '@redhat-cloud-services/frontend-components/useChrome';
 import ClusterImagesTable from './ClusterImagesTable';
-import { CLUSTER_DETAIL_TABS } from '../../../Helpers/constants';
+import {
+  CLUSTER_DETAIL_TABS,
+  EXPOSED_IMAGES_FEATURE_FLAG,
+} from '../../../Helpers/constants';
 import TableTabs from '../../PresentationalComponents/TableTabs';
+import { useFeatureFlag } from '../../../Helpers/hooks';
 
 const ClusterDetailPage = ({ activeTab }) => {
   const chrome = useChrome();
 
+  const areExposedImagesEnabled = useFeatureFlag(EXPOSED_IMAGES_FEATURE_FLAG);
```

Comment:
nit-pic: We could also wrap the whole /cve/:cveId/image route under feature flag as well to avoid blank screens/tabs.

Reply:
We could, I was thinking about this, but we do not plan to disable the feature flag after it's been enabled so no one can really have hidden tabs in history/bookmarks.